.pre-commit-config.yaml

```diff
@@ -2,14 +2,14 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.6.0
```

Comment:
Please don't upgrade .pre-commit-config file. This is being overwrriten by bot creating a unnecessary tag/release.

Reply:
Can you point me the the guide about the process of updating the hooks? In the pre-commit files it is written to get latest release from here  `# Get the latest from: https://github.com/antonbabenko/pre-commit-terraform/releases`

Reply:
Here is an example 
1. Your [PR](https://github.com/honestbank/terraform-gcp-gcs/pull/26) updated the pre-commit file 
2. Your changes to pre-commit file were overwritten by [sync PR. ](https://github.com/honestbank/terraform-gcp-gcs/pull/27)

Either update the pre-commit file in .github sync or please don't make changes. Additionally, please create a doc on updating the pre-commit hooks if it doesn't exist.

Reply:
Approving this PR to unblock you and will take up this discussion in the stand up or any team meeting.

Reply:
@bibek4699 this repository should be included in the [list of public Terraform repositories](https://github.com/honestbank/.github/blob/main/.github/sync-other-files.yml#L275-L294) that have sync files from our `.github` repo, so that it is kept up-to-date.